plaso/parsers/sqlite.py

```diff
@@ -418,6 +426,20 @@ def ParseFileEntry(self, parser_mediator, file_entry):
         if not plugin.REQUIRED_TABLES.issubset(table_names):
           continue
 
+        columns_match = True
+        for required_table, required_columns in plugin.REQUIRED_COLUMNS.items():
+          if required_table not in table_names:
+            columns_match = False
+            continue
+
+          if not frozenset(required_columns).issubset(
+              database.table_columns.get(required_table)):
+            columns_match = False
+            continue
```

Comment:
same here, why loop over remaining tables and columns if you know the outcome is a fail.